StripeFinancialConnections/StripeFinancialConnections/Source/Common/HostController.swift

```diff
@@ -170,6 +163,26 @@ private extension HostController {
         webFlowViewController.delegate = self
         navigationController.setViewControllers([webFlowViewController], animated: true)
     }
+
+    func continueWithNativeFlow(_ synchronizePayload: FinancialConnectionsSynchronize) {
```

Comment:
thats nice that its similar to `continueWithWebFlow` now (separate parts) - separation is not as important as just consistency